OpenRA.Mods.Common/Traits/Buildings/Reservable.cs

```diff
@@ -64,21 +65,25 @@ public static bool IsReserved(Actor a)
 			return res != null && res.reservedForAircraft != null && !res.reservedForAircraft.MayYieldReservation;
 		}
 
-		public void Disposing(Actor self)
+		void INotifyActorDisposing.Disposing(Actor self)
 		{
 			if (reservedForAircraft != null)
 				reservedForAircraft.UnReserve();
 		}
 
-		public void OnOwnerChanged(Actor self, Player oldOwner, Player newOwner)
+		void INotifyOwnerChanged.OnOwnerChanged(Actor self, Player oldOwner, Player newOwner)
 		{
 			if (reservedForAircraft != null)
 				reservedForAircraft.UnReserve();
 		}
 
-		public void Selling(Actor self) { Sold(self); }
+		void INotifySold.Selling(Actor self)
+		{
+			if (reservedForAircraft != null)
```

Comment:
This block is now duplicated four times.  Can you please define a private `UnReserve()` method they all call?